dashboard/dashboard.py

```diff
@@ -206,19 +198,35 @@ def copy_setup(experiments_dir, setup_dir, exp_name):
 
 
 def run_experiment(info, experiments_dir, tmp_data_dir, exp_name):
+
+    to_local_time = lambda sec: time.asctime(time.localtime(sec))
     exp_dir = os.path.join(experiments_dir, exp_name)
     exp_conf = info.exp_config_dir(exp_name)
 
     # set up a temporary data directory for that experiment
     exp_data_dir = os.path.join(tmp_data_dir, exp_name)
     idemp_mkdir(exp_data_dir)
 
+    # Mark the start and the end of an experiment
+    start_time = time.time()
+    start_msg = f'Experiment {exp_name} starts @ {to_local_time(start_time)}'
+    print_log(start_msg)
     # run the run.sh file on the configs directory and the destination directory
     subprocess.call([os.path.join(exp_dir, 'run.sh'), exp_conf, exp_data_dir],
                     cwd=exp_dir)
-
+    end_time = time.time()
+    delta = datetime.timedelta(seconds=end_time - start_time)
     # collect the status file from the destination directory, copy to status dir
     status = validate_status(exp_data_dir)
+    # write time information if the experiment succeed
```

Comment:
I think it would be useful to record times even if it fails (knowing how long it took to fail can be informative too)

Reply:
Sure! It's a good idea. I've made the information available regardless of whether the experiments are successful or not.